### PR TITLE
Docs:  add parameters to /config

### PR DESCRIPTION
**Pull request by bkmartinjr**
Proposed REST changes to add parameter communication from server to client.

### CODE REVIEW

File: docs/REST_API.md [hunk 1]
```diff
@@ -230,6 +233,12 @@ GET /config
       "engine": "ScanPy version 1.33",
       "dataset": "/home/joe/mouse/blorth.csv"
     },
+    "parameters": {
+      "verbose": false,
+      "log-level": "info",
+      "minimumThreshold", 99
```
Comment (csweaver) [inline docs/REST_API.md]: why is this minimum and not maximum?

File: docs/REST_API.md [hunk 2]
```diff
@@ -230,6 +233,12 @@ GET /config
       "engine": "ScanPy version 1.33",
       "dataset": "/home/joe/mouse/blorth.csv"
     },
+    "parameters": {
+      "verbose": false,
+      "log-level": "info",
```
Comment (csweaver) [inline docs/REST_API.md]: Where are verbose and log-level coming from? Is this the `--debug` param? If so, what are our log-levels? We should probably define them.

Comment (bkmartinjr) [reply docs/REST_API.md]: they are just examples - intended to show what _might_ be.

you are right that when we add our first parameter, we need to document it.  Probably need to add a section that describes  each.   Given that we do not yet have any, I made up some examples.

File: docs/REST_API.md (hunk 1 above)
Comment (bkmartinjr) [reply docs/REST_API.md]: just another random example...could be anything.

File: docs/REST_API.md (hunk 2 above)
Comment (bkmartinjr) [reply docs/REST_API.md]: BTW, we could hold this PR (and add to it) once we have our first set of CLI params defined.

Comment (csweaver) [reply docs/REST_API.md]: Yeah, let's do that. The doc looks good outside of the example params. Technically they are fine, but I'm worried they would be confusing to a new contributor who thinks they are required.

**Comment by csweaver** [comment]
If there are no parameters to pass through, is there still an empty "parameters" key? 
```
                "parameters" : { }              
```

**Comment by bkmartinjr** [comment]
Yes, it should be an empty parameters object if no params

**Pull request merged**